exporter/streams.go

```diff
@@ -111,6 +115,14 @@ func scanStreamGroupConsumers(c redis.Conn, stream string, group string) ([]stre
 	return result, nil
 }
 
+func ParseId(id string) float64 {
```

Comment:
Maybe give this a little more descriptive name like `parseStreamId`?

Reply:
changed to `parseStreamItemID` in 70664ce